### PR TITLE
cloud: add release notes for 20221011

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

As title.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### CODE REVIEW

File: tidb-cloud/release-notes-2022.md [hunk 1]
```diff
@@ -8,6 +8,19 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## October 11, 2022
+
+**General changes**
+
+* Upgrade the default TiDB version of new Developer Tier clusters from v6.2.0 to v6.3.0.
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:15]: Please add links to Developer Tier, v6.2.0, and v6.3.0.

File: tidb-cloud/release-notes-2022.md [hunk 2]
```diff
@@ -8,6 +8,19 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+**Console changes**
+
+* Optimize billing information in [Bills](/tidb-cloud/tidb-cloud-billing.md#billing-details):
+
+    * Provide more fine-grained billing information at the node level in the Summary by service section.
+    * Add a Usage details section. You can also download usage details as a CSV file.
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:22]:
```suggestion
    * Add a usage details section. You can also download usage details as a CSV file.
```

File: tidb-cloud/release-notes-2022.md [hunk 3]
```diff
@@ -8,6 +8,19 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
 
+## October 11, 2022
+
+**General changes**
+
+* Upgrade the default TiDB version of new Developer Tier clusters from v6.2.0 to v6.3.0.
+
+**Console changes**
+
+* Optimize billing information in [Bills](/tidb-cloud/tidb-cloud-billing.md#billing-details):
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:19]:
```suggestion
* Optimize billing information in the [billing details page](/tidb-cloud/tidb-cloud-billing.md#billing-details):
```

File: tidb-cloud/release-notes-2022.md [hunk 4]
```diff
@@ -8,6 +8,19 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    * Provide more fine-grained billing information at the node level in the Summary by service section.
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:21]:
```suggestion
    * Provide more fine-grained billing information at the node level in the **Summary By Service** section.
```

File: tidb-cloud/release-notes-2022.md (hunk 2 above)
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:22]:
```suggestion
    * Add a **Usage Details** section. You can also download usage details as a CSV file.
```

File: tidb-cloud/release-notes-2022.md (hunk 1 above)
Comment (ran-huang) [inline tidb-cloud/release-notes-2022.md:15]:
```suggestion
* Upgrade the default TiDB version of new [Developer Tier](/tidb-cloud/select-cluster-tier.md#developer-tier) clusters from [v6.2.0](https://docs.pingcap.com/tidb/v6.2/release-6.2.0) to [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0).
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: tidb-cloud/release-notes-2022.md [hunk 5]
```diff
@@ -8,6 +8,19 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
 
+## October 11, 2022
+
+**General changes**
+
+* Upgrade the default TiDB version of new [Developer Tier](/tidb-cloud/select-cluster-tier.md#developer-tier) clusters from [v6.2.0](https://docs.pingcap.com/tidb/v6.2/release-6.2.0) to [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0).
+
+**Console changes**
+
+* Optimize billing information in the [billing details page](/tidb-cloud/tidb-cloud-billing.md#billing-details):
```
Comment (hfxsd) [inline tidb-cloud/release-notes-2022.md:19]:
```suggestion
* Optimize billing information on the [billing details page](/tidb-cloud/tidb-cloud-billing.md#billing-details):
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**